### DISCUSSION TITLE
camelCase field name not much readable in admin UI when transformed to UPPERCASE

### DISCUSSION THREAD

**Discussion by babakfp** [pocketbase/pocketbase#1779] [Q&A] [answered]
Hey

![Screenshot 2023-02-05 020740](https://user-images.githubusercontent.com/44144724/216812981-ed1affa5-fc88-43af-96b6-bc0ce2706893.png)
![Screenshot 2023-02-05 020750](https://user-images.githubusercontent.com/44144724/216812982-9e0912c6-6fd0-404f-9ba2-a99f3bd3da64.png)

I like skane_case more than camelCase but for consistency reasons (between native functions) I'm using camelCase.

---

I just did a quick google search and realized I have a typo (it should be Tier not Tear). now I need to rename a bunch of stuff😂

**Comment by ganigeorgiev** [comment, marked as answer]
All field labels are uppercased.

I'll consider eventually to add an exception for the collection fields and to display the original casing, but this is a very low priority for now.
